src/Components/Cart.js

```diff
@@ -0,0 +1,61 @@
+import bonusItems from "../data/bonusItems";
+export default function Cart({ cart, setCart}) {
+  let cartTotal = cart.reduce((a, b) => {
+    return b.amount + a;
+  }, 0);
+
+  //removing bird from cart
+  const updateBirdsArray = (bird) => {
+    cart.splice(cart.indexOf(bird), 1);
```

Comment:
We should always make a copy of the state ([...cart]) before we make changes to it. Splice changes the array, so we need a copy first